README.md

```diff
@@ -74,6 +74,47 @@ Be sure to use Python 3, as it requires a function from the socket module
 that's only available in this version (wrapper around if_nametoindex(3)).
 
 
+## Using net-setup.sh script to setup host side ethernet interface
+
+The net-setup.sh script can setup an ethernet interface to the host.
+User is able to setup a configuration file that will contain
+commands to setup IP addresses and routes to the host interface.
+This net-setup.sh script will need to be run as a root user.
```

Comment:
root access is not necessary. cap_net_admin capability could suffice. For example after running this command root is not needed.

`sudo setcap cap_net_admin=eip ./net-setup.sh`

getcap can be used to check the capabilities

Reply:
Yep, I will update the documentation